api/services/rag_pipeline/rag_pipeline_transform_service.py

```diff
@@ -149,8 +149,7 @@ def _deal_file_extensions(self, node: dict):
         file_extensions = node.get("data", {}).get("fileExtensions", [])
         if not file_extensions:
             return node
-        file_extensions = [file_extension.lower() for file_extension in file_extensions]
-        node["data"]["fileExtensions"] = DOCUMENT_EXTENSIONS
+        node["data"]["fileExtensions"] = [ext.lower() for ext in file_extensions if ext in DOCUMENT_EXTENSIONS]
```

Comment:
cc @QuantumGhost @JohnJyong 
Is this as expected for filtering extentions without throwing exceptions?